### DISCUSSION TITLE
Trying to add Jetbrains mono nerdfont - getting confused...

### DISCUSSION THREAD

**Discussion by victorhooi** [dustinlyons/nixos-config#88] [Q&A]
I'm trying to install a font (Jetbrains Mono with Nerdfonts, so I can use starship.rs), and I'm going around in circles trying to figure out the correct way to do this...

Firstly, I can't seem to find the patched Jetbrains Mono font with Nerdfonts on Nix packages:

https://search.nixos.org/packages?channel=23.11&show=nerdfonts&from=0&size=50&sort=relevance&type=packages&query=nerdfont

There only seem to be a couple of fonts that have been patched, and offered as packages.

Is the solution to install the font via Homebrew?

https://formulae.brew.sh/cask/font-jetbrains-mono-nerd-font#default

Then secondly, I'm not really sure of the proper way to configure it.

The Nix wiki says that you need to use `fonts.packages`:

https://nixos.wiki/wiki/Fonts

But then [this article](https://nixos.asia/en/tips/hm-fonts) mentioned you can simply use home-manager, add it to the home-manager packages (which I think in your template are done via normal nix packages?), and enable fontconfig?

```
{
  home.packages = [
    # Fonts
    cascadia-code
  ];

  fonts.fontconfig.enable = true;
}
```

(But of course, there's no nix package for the nerdfonts?)

What's the proper way I should be doing this?

**Comment by RetroTypeOfAGuy** [comment]
this is fro my config 
fonts = {
    enableDefaultPackages = true;
    fontconfig.enable = true;
    packages = with unstable; [
      nerd-fonts.jetbrains-mono
    ];
  };
nix-24.11
